amundsen_application/static/js/components/TableDetail/EditableSection/tests/index.spec.tsx

```diff
@@ -65,5 +66,18 @@ describe("EditableSection", () => {
       const { wrapper } = setup(null, child);
       expect(wrapper.childAt(1).text()).toBe(child);
     });
+
+    it("renders button when readOnly=false", () => {
+      expect(wrapper.find(".btn").length).toEqual(1);
```

Comment:
```suggestion
      expect(wrapper.find(".edit-button").length).toEqual(1);
```
All buttons in this application have the class `.btn`, so I'm suggesting the className specifically for the edit button in case there is ever another button.